README.adoc

```diff
@@ -1,28 +1,17 @@
 :projectKey: optaweb-employee-rostering
-:sonarBadge: image:https://sonarcloud.io/api/project_badges/measure?project={projectKey}
-:sonarLink: link="https://sonarcloud.io/dashboard?id={projectKey}"
+
+*This project is no longer maintained.*
+Visit https://github.com/kiegroup/optaplanner-quickstarts/[OptaPlanner Quickstarts] to see how to integrate https://www.optaplanner.org/[OptaPlanner] in your application.
```

Comment:
Maybe direct them directly to the employee scheduling quickstart (https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/employee-scheduling )